service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisOtherParty.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class DocmosisOtherParty {
```

Comment:
There seems to be a lot of similarities between this, Child, and Respondent. Any chance there could be a super class that these extend that group the common elements

Reply:
looks like something for a another pr. thoughts?

Reply:
Could do, you are affecting all the classes already in this one so it wouldn't be extra files changed (apart from the super class), although you would then have to check that SDO and CMO aren't adversly effected so I guess it will be fine for another PR